compiler-cli/templates/gleam@@compile.erl

```diff
@@ -150,7 +150,11 @@ add_lib_to_erlang_path(Lib) ->
     code:add_paths(filelib:wildcard([Lib, "/*/ebin"])).
 
 del_lib_from_erlang_path(Lib) ->
-    code:del_paths(filelib:wildcard([Lib, "/*/ebin"])).
+    Paths = filelib:wildcard([Lib, "/*/ebin"]),
+    case erlang:function_exported(code, del_paths, 1) of
```

Comment:
Let's do this using conditional compilation, to avoid doing the lookup multiple times.

```erl
-if(?OTP_RELEASE > 25).
    code:del_paths(Paths).
-else.
    lists:foreach(fun code:del_path/1, Paths).
-endif.
```